packages/camera/camera/android/src/main/java/io/flutter/plugins/camera/features/sensororientation/DeviceOrientationManager.java

```diff
@@ -142,26 +142,32 @@ public int getPhotoOrientation(PlatformChannel.DeviceOrientation orientation) {
   }
 
   /**
-   * Returns the device's video orientation in degrees based on the sensor orientation and the last
-   * known UI orientation.
+   * Returns the device's video orientation in clockwise degrees based on the sensor orientation and
```

Comment:
I'm not sure if I should mention that this is for both front and back-facing cameras, because the documentation I  referred to didn't specifically mention that. I'd rather only document something of which there is actual proof that can be backed up by the Android docs.